zhaquirks/tuya/tuya_motion.py

```diff
@@ -1610,3 +1610,71 @@ class TuyaMotionDetectionMode(t.enum8):
     .skip_configuration()
     .add_to_registry()
 )
+
+
+# Mercator Ikuü TS0601 motion sensor (_TZE200_agumlajc)
+class LightControlMode(t.enum8):
+    """Enum for the light control mode setting (dp_id=105)."""
+
+    On = 0
+    Off = 1
+    Auto = 2
+
+
+(
+    TuyaQuirkBuilder("_TZE200_agumlajc", "TS0601")
+    .applies_to("_TZE204_agumlajc", "TS0601")
+    .tuya_temperature(dp_id=1, scale=10)
+    .tuya_humidity(dp_id=2)
+    .tuya_illuminance(dp_id=101)
+    .tuya_number(
+        dp_id=102,
+        attribute_name="brightness_threshold",
+        type=t.uint16_t,
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        min_value=0,
+        max_value=1000,
+        step=1,
+        translation_key="brightness_threshold",
+        fallback_name="Brightness threshold",
+    )
```

Comment:
Can you also quickly explain what this entity does? Is motion not detected if it's too dark or bright?